### PR TITLE
Add a test for 1259

### LINKED ISSUE #1259: OVA does not work if features column is not "Features"
### System information

- **Windows**: 10.0.17134
- **.NET Version (eg., dotnet --info)**: 2.1.402

### Issue

OVA still looks for columns "Features" and cannot find it in the Following test. 

### Source code / logs

        [Fact]
        public void New_MetacomponentsFeaturesRenamed()
        {
            using (var env = new LocalEnvironment())
            {
                var data = new TextLoader(env, MakeIrisTextLoaderArgs())
                    .Read(new MultiFileSource(GetDataPath(TestDatasets.irisData.trainFilename)));

                var sdcaTrainer = new LinearClassificationTrainer(env, "Vars", "Label", advancedSettings: (s) => { s.MaxIterations = 100; s.Shuffle = true; s.NumThreads = 1; });
                var pipeline = new ConcatEstimator(env, "Vars", "SepalLength", "SepalWidth", "PetalLength", "PetalWidth")
                    .Append(new TermEstimator(env, "Label"), TransformerScope.TrainTest)
                    .Append(new Ova(env, sdcaTrainer))
                    .Append(new KeyToValueEstimator(env, "PredictedLabel"));

                var model = pipeline.Fit(data);
            }
        }

### PR DESCRIPTION
**Pull request by sfilipi**
Closes #1259 by adding the test described in the bug: OVA with the feature column named something other than "Features". 

A bit of cleanup on reusing arguments to load the test dataset.

### CODE REVIEW

File: test/Microsoft.ML.TestFramework/Datasets.cs [hunk 1]
```diff
@@ -175,7 +179,18 @@ public static class TestDatasets
         {
             name = "sentiment",
             trainFilename = "wikipedia-detox-250-line-data.tsv",
-            testFilename = "wikipedia-detox-250-line-test.tsv"
+            testFilename = "wikipedia-detox-250-line-test.tsv",
+            GetLoaderColumns= () =>
+            {
```
Comment (sfilipi) [inline test/Microsoft.ML.TestFramework/Datasets.cs]: fix

File: test/Microsoft.ML.TestFramework/Datasets.cs [hunk 2]
```diff
@@ -175,7 +179,15 @@ public static class TestDatasets
...
+                return new[]
+                {
+                    new TextLoader.Column("Label", DataKind.BL, 0),
+                    new TextLoader.Column("SentimentText", DataKind.Text, 1)
+                }; ;
```
Comment (artidoro) [inline test/Microsoft.ML.TestFramework/Datasets.cs]: >; ; [](start = 17, length = 3)

Remove the repeated semicolon.

File: test/Microsoft.ML.TestFramework/Datasets.cs [hunk 3]
```diff
@@ -175,7 +179,15 @@ public static class TestDatasets
         {
             name = "sentiment",
             trainFilename = "wikipedia-detox-250-line-data.tsv",
-            testFilename = "wikipedia-detox-250-line-test.tsv"
+            testFilename = "wikipedia-detox-250-line-test.tsv",
+            GetLoaderColumns= () =>
```
Comment (artidoro) [inline test/Microsoft.ML.TestFramework/Datasets.cs]: >= [](start = 28, length = 1)

Add space

**Comment by sfilipi** [comment]
@justinormont thanks for pointing it out. Updating the description. 

---
In reply to: [446049496](https://github.com/dotnet/machinelearning/pull/1849#issuecomment-446049496) [](ancestors = 446049496)
